packages/store/src/legacyWeakMap.js

```diff
@@ -1,50 +1,40 @@
 // Copyright (C) 2019 Agoric, under Apache license 2.0
 
-// @ts-check
```

Comment:
Why remove this?

Reply:
I have no idea, but it's back. Resolving.